### ISSUE TITLE
Can't install on Windows 

### ISSUE THREAD

**Issue by Miwwa** [cayasso/cacheman-redis#7]
I have Windows 7 and I can't install module with `npm install cacheman-redis`
Please, fix it.

```
npm WARN optional dep failed, continuing fsevents@1.0.1
|
> cacheman-redis@1.0.0 postinstall D:\Dropbox\Projects\MyProject\node_modules\cacheman-redis
> make

"make" не является внутренней или внешней
командой, исполняемой программой или пакетным файлом.
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "cacheman-redis" "--save"
npm ERR! node v4.1.1
npm ERR! npm  v2.14.4
npm ERR! code ELIFECYCLE

npm ERR! cacheman-redis@1.0.0 postinstall: `make`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cacheman-redis@1.0.0 postinstall script 'make'.
npm ERR! This is most likely a problem with the cacheman-redis package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     make
npm ERR! You can get their info via:
npm ERR!     npm owner ls cacheman-redis
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\Dropbox\Projects\MyProject\npm-debug.log
```

npm debug log - https://dl.dropboxusercontent.com/u/46327478/npm-debug.log

**Comment by evenchange4** [comment]
+1 on MAC

**Comment by appleboy** [comment]
fix on #8

**Comment by cayasso** [comment]
Fixed by this commit https://github.com/cayasso/cacheman-redis/commit/d0f6a019d6f530c6204fd7243c487c053b934b15#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R9
